ui/gulpfile.js

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const gulp = require('gulp');
+const path = require('path');
+const cp = require('child_process');
+
+gulp.task('test', (cb) => {
```

Comment:
Curious why you switched to gulp?

Reply:
1. I find it easier to work with gulp than scripts. For example I was trying to pass in "CODE_EXTENSIONS_PATH" as "./test/extension", but it wasn't working unless I passed in the _full_ path. Wasn't sure how to do that for a script, but it's easier to do (and debug) in javascript (with `path.join(__dirname ...`)
1. I also just find it more readable. Scripts start to get out of hand when you have multiple environment variables.